fastlane/docs/Plugins.md

```diff
@@ -30,6 +30,28 @@ What will this do
 - Make sure your `fastlane/Pluginfile` is properly referenced from your `./Gemfile`
 - Run `bundle install` to make sure all required dependencies are installed on your local machine (this step might ask for your admin password to install Ruby gems)
 
+#### Plugin Source
+
+Your `fastlane/Pluginfile` contains the list of all `fastlane` plugins your project uses. The `Pluginfile` is a [Gemfile](http://bundler.io/gemfile.html) that gets imported from your main `Gemfile`.
+You specificy all dependencies, including the required version numbers:
+
+```ruby
+# Fetched from RubyGems.org
+gem "fastlane-plugin-xcversion"
+
+# Fetched from GitHub
+gem "fastlane-plugin-xcversion", git: "https://github.com/fastlane/fastlane-plugin-xcversion"
+
+# Fetched from a local directory
+gem "fastlane-plugin-xcversion", path: "https://github.com/fastlane/fastlane-plugin-xcversion"
+
+# Specify a version requirements
+gem "fastlane-plugin-xcversoin", "1.1.0"
+gem "fastlane-plugin-xcversoin", ">= 1.0"
```

Comment:
Thank you! Actually helped solving my other issue. :o)

> On 25 May 2016, at 17:37, James Campbell <notifications@github.com> wrote:
> 
> In fastlane/docs/Plugins.md <https://github.com/fastlane/fastlane/pull/4722#discussion_r64595282>:
> 
> > +Your `fastlane/Pluginfile` contains the list of all `fastlane` plugins your project uses. The `Pluginfile` is a [Gemfile](http://bundler.io/gemfile.html) that gets imported from your main `Gemfile`.
> > +You specificy all dependencies, including the required version numbers:
> > +
> > +```ruby
> > +# Fetched from RubyGems.org
> > +gem "fastlane-plugin-xcversion"
> > +
> > +# Fetched from GitHub
> > +gem "fastlane-plugin-xcversion", git: "https://github.com/fastlane/fastlane-plugin-xcversion"
> > +
> > +# Fetched from a local directory
> > +gem "fastlane-plugin-xcversion", path: "https://github.com/fastlane/fastlane-plugin-xcversion"
> > +
> > +# Specify a version requirements
> > +gem "fastlane-plugin-xcversoin", "1.1.0"
> > +gem "fastlane-plugin-xcversoin", ">= 1.0"
> Should be xcversion :P
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub <https://github.com/fastlane/fastlane/pull/4722/files/9525407571e7fecf14605476e40d42bbe6dcb463..e4328092a37bb0a9a188c2075d7349dda808ef48#r64595282>